javatests/arcs/core/entity/HandleManagerTestBase.kt

```diff
@@ -549,22 +549,22 @@ open class HandleManagerTestBase {
             assertThat(handle.size()).isEqualTo(2)
             assertThat(handle.isEmpty()).isEqualTo(false)
             assertThat(handle.fetchAll()).containsExactly(entity1, entity2)
-            assertThat(handle.fetchById(entity1.entityId)).isEqualTo(entity1)
-            assertThat(handle.fetchById(entity2.entityId)).isEqualTo(entity2)
+            assertThat(handle.dispatchFetchById(entity1.entityId)).isEqualTo(entity1)
+            assertThat(handle.dispatchFetchById(entity2.entityId)).isEqualTo(entity2)
```

Comment:
Actually not needed for this test; this block is already running on the dispatcher (see line 535).

Reply:
Hmm, I'd agree but the test failed without them on my machine.
Any ideas as to why?

Reply:
I would be very surprised and quite worried if calling `fetchById` inside a dispatcher context block caused problems, given that this is all the dispatch version of the function does...

Reply:
Yeah, can confirm that these ones were fine. I was a little overzealous with the `find and replace`.